.chronus/changes/prevent-carry-over-2024-2-13-14-34-30.md

```diff
@@ -0,0 +1,7 @@
+---
+changeKind: fix
+packages:
+  - "@azure-tools/typespec-client-generator-core"
+---
+
+prevent carry over for @clientName
```

Comment:
```suggestion
prevent carry over for `@clientName`
```
make sure you wrap decorator as this is markdown and in github it will tag the user of that name otherwise